### DISCUSSION TITLE
Configuration CrossDomains options

### DISCUSSION THREAD

**Discussion by Pavel-Mishalov** [OvenMediaLabs/OvenMediaEngine#1557] [Help]
hi
I have an api service configured with Server->Managers->API->CrossDomains
and configure settings Applications->Application->Publishers->LLHLS->CrossDomains->Url = *

I expected that all api requests would be blocked by the list of available addresses, and the playlist would be public to everyone. But the playlist request **/app/m/llhls.m3u8** always crashes due to a cors error in the browser. If you request a playlist using curl or go to it via a direct link, the cors errors disappear and the playlist becomes available

[Server.xml.txt](https://github.com/AirenSoft/OvenMediaEngine/files/14625657/Server.xml.txt)

**Comment by getroot** [comment]
Could you capture the browser screen with dev console window when you get the cors error?
